server/src/main/java/de/pnp/manager/network/ClientHandler.java

```diff
@@ -159,6 +161,24 @@ public StringProperty clientNameProperty() {
         return controlledCharacters;
     }
 
+    @Override
+    public Collection<String> getAccessibleInventories() {
+        return accessibleInventories;
+    }
+
+    @Override
+    public boolean hasAccessToInventory(String id) {
+        return controlledCharacters.contains(id) || accessibleInventories.contains(id);
+    }
+
+    private String getSessionID() {
+        return getCurrentSession() != null ? getCurrentSession().getSessionID() : null;
+    }
+
+    private Collection<Client> getClientsWithInventoryAccess(String id) {
+        return manager.getNetworkHandler().clientsProperty().stream().filter(c -> c.hasAccessToInventory(id)).collect(Collectors.toList());
+    }
```

Comment:
why is this a method in ClientHandler

Reply:
Because it is a helper function to reduce code duplication and it is not needed anywhere else at the current moment